lib/scripts/k2s/start/Start.ps1

```diff
@@ -328,6 +328,8 @@ route delete $clusterCIDRServicesWindows >$null 2>&1
 Write-Log "Add route to $clusterCIDRServicesWindows"
 route -p add $clusterCIDRServicesWindows $ipControlPlane METRIC 7 | Out-Null
 
+Restart-NlaSvc
```

Comment:
Can we do this when HNS, vmcompute services are restarted? Is there a particular significance doing it here? Same goes to Start script as well.
https://github.com/Siemens-Healthineers/K2s/blob/488681bb7a79f3a808be0d9f91208bbc67a96a6f/lib/scripts/k2s/start/Start.ps1#L193

Reply:
The idea was to do it anytime before BeforeStartK8sNetwork hook. It can be moved also to be restarted with HNS and vmcompute